src/char/barbarian.py

```diff
@@ -77,21 +75,19 @@ def _do_hork(self, hork_time: int):
                 Logger.warning("Turns out weapon switch just took a long time. You ever considered getting a new internet provider or to upgrade your pc?")
 
     def pre_buff(self):
-        # keyboard.send(self._char_config["weapon_switch"])
```

Comment:
These pre buff weapon switch probably shouldn't have changed with this pr. These are unrelated to hork time setting below. Remove them from this pr.